lib/miq_automation_engine/service_models/miq_ae_service_container_template.rb

```diff
@@ -0,0 +1,8 @@
+module MiqAeMethodService
+  class MiqAeServiceContainerTemplate < MiqAeServiceModelBase
+    expose :container_template_parameters, :association => true
+    expose :ext_management_system,         :association => true
+    expose :is_tagged_with?
+    expose :tags
```

Comment:
I realized these tag methods got into a lot of the container models, but they need to be removed.  You should be using the tag methods defined for at the base service model level here: 
 https://github.com/ManageIQ/manageiq/blob/master/lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_model_base.rb#L218

Will need to open a git issue to clean up the other service models that added these methods:
miq_ae_service_container.rb
miq_ae_service_container_build.rb
miq_ae_service_container_build_pod.rb
miq_ae_service_container_deployment_node.rb
miq_ae_service_container_group.rb
miq_ae_service_container_image.rb
miq_ae_service_container_image_registry.rb
miq_ae_service_container_node.rb
miq_ae_service_container_replicator.rb
miq_ae_service_container_route.rb
miq_ae_service_container_service.rb
miq_ae_service_container_volume.rb